pkg/component/crddeployer/crd_test.go

```diff
@@ -59,6 +101,16 @@ var _ = Describe("CRD", func() {
 kind: CustomResourceDefinition
 metadata:
     name: myresources.mygroup.example.com`
+		confirmationCRDString = `apiVersion: apiextensions.k8s.io/v1
```

Comment:
One minor nit (after all, also `crd1`, etc. is a ´string` 😉 ):

```suggestion
		confirmationCRD = `apiVersion: apiextensions.k8s.io/v1
```

Reply:
What's the difference to the already existing `crd1Ready`? Maybe we can reuse that one here?

Reply:
`confirmationCRD` is already a `CustomResourceDefinition` object defined earlier.
They are actually the same right now 😆 
I forgot to put the deletion protection on it :/

Thank you very much